package.json

```diff
@@ -5,10 +5,10 @@
   "scripts": {
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "coverage-html-report": "rm -rf coverage/ && nyc report --reporter=html && opn coverage/index.html",
-    "dependency-check": "dependency-check package.json --missing --unused -i dependency-check -i nyc -i opn-cli -i standard -i napi-macros -i coveralls --entry index.js --entry test/index.js --entry scripts/*.js",
+    "dependency-check": "dependency-check package.json --missing --unused -i dependency-check -i version-changed -i nyc -i opn-cli -i standard -i napi-macros -i coveralls --entry index.js --entry test/index.js --entry scripts/*.js",
     "generate-constants": "./scripts/generate-constants.js",
     "install": "node-gyp-build scripts/rebuild-core.js",
-    "prebuild": "node scripts/prebuildify.js",
+    "prebuild": "version-unchanged || node scripts/prebuildify.js",
     "submodule": "git submodule update --recursive --init",
```

Comment:
So when `version-unchanged` is _successful_ we will stop there. If it _fails_ we will do the other branch, e.g. the prebuildify stuff.